From aaaba10e783eadc965166394e2c3e3e86d3ef2ea Mon Sep 17 00:00:00 2001
From: Hec <hec@heccraft.com>
Date: Sat, 26 Sep 2026 23:14:50 -0400
Subject: [PATCH] Fix deps

This is needed since we're overriding a lot of GNOME elements with FDSDK
elements, and the runtime dependencies of a few of them (e.g glib) are
different.
---
 elements/core-deps/gnome-online-accounts.bst | 1 +
 elements/core-deps/librest.bst               | 1 +
 2 files changed, 2 insertions(+)

diff --git a/elements/core-deps/gnome-online-accounts.bst b/elements/core-deps/gnome-online-accounts.bst
index d739d35ba..b9d6e129d 100644
--- a/elements/core-deps/gnome-online-accounts.bst
+++ b/elements/core-deps/gnome-online-accounts.bst
@@ -14,6 +14,7 @@ build-depends:
 - sdk/vala.bst
 - freedesktop-sdk.bst:components/docbook-xsl.bst
 - freedesktop-sdk.bst:components/libxslt.bst
+- freedesktop-sdk.bst:components/krb5.bst
 
 depends:
 - core-deps/librest.bst
diff --git a/elements/core-deps/librest.bst b/elements/core-deps/librest.bst
index e3fd02a1f..82e3e04a4 100644
--- a/elements/core-deps/librest.bst
+++ b/elements/core-deps/librest.bst
@@ -11,6 +11,7 @@ build-depends:
 - buildsystems/meson.bst
 - sdk/gi-docgen.bst
 - sdk/gobject-introspection.bst
+- sdk/sysprof-minimal.bst
 
 depends:
 - sdk/glib.bst
-- 
2.56.0

